### PR TITLE
Adding MAX_TARGET_COVERAGE to targeted metrics

### PR DESCRIPTION
**Pull request by nh13**
Also adds the metric to HsMetrics and TargetedPcrMetrics.

### CODE REVIEW

**Comment by nh13** [comment]
@yfarjoun could you take a quick look at this?

**Comment by nh13** [comment]
@gbggrant thanks for taking a look at this, it should be quick!

File: src/main/java/picard/analysis/directed/HsMetrics.java [hunk 1]
```diff
@@ -121,6 +121,9 @@
     /** The median coverage of a target region. */
     public double MEDIAN_TARGET_COVERAGE;
 
+    /** The maximum coverage of reads that mapped to target regions of an experiment. */
+    public double MAX_TARGET_COVERAGE;
```
Comment (gbggrant) [inline src/main/java/picard/analysis/directed/HsMetrics.java]: Could be long instead of double.

Comment (nh13) [inline src/main/java/picard/analysis/directed/HsMetrics.java]: Copy-paste, sorry, will fix, then merge after tests.

**Pull request merged**